lib/translate.go

```diff
@@ -34,6 +34,8 @@ import (
 //     // item's config field
 //     "widgets.config": "",
 //   })
+//
+// Deprecated: Use lib/decode.HookTranslateKeys instead.
```

Comment:
There is one place in the enterprise code thats still using this too. Otherwise I think this would be dead code and we could just get rid of it.

Reply:
I've left the CA provider use of this in OSS as well. I'm much less familiar with that piece of code so I wanted to split it out into a separate change. I expect we will be able to do the same thing.